planet/planetsympy/config

```diff
@@ -365,3 +365,9 @@ feed 15m http://sartajsingh.me/feeds/sympy.xml
         define_face hackergotchi/leosartaj.jpg
         define_facewidth 80
         define_faceheight 80
+
+feed 15m http://s-hacker.info/blog/gsoc16/feeds/sympy.xml
```

Comment:
I am trying to fix this, but I am getting problem. It will be helpful if someone tells me a good way to get .xml file of any webpage for RSS.